pl_bolts/callbacks/vision/image_generation.py

```diff
@@ -30,9 +30,11 @@ class TensorboardGenerativeModelImageSampler(Callback):
         trainer = Trainer(callbacks=[TensorboardGenerativeModelImageSampler()])
     """
 
-    def __init__(self, num_samples: int = 3):
+    def __init__(self, num_samples: int = 3, nrow: int = 1, normalize: bool = True):
         super().__init__()
         self.num_samples = num_samples
+        self.nrow = nrow
+        self.normalize = normalize
```

Comment:
Shall we simplify this PR by reverting the change here as it is being discussed in #489? Without this change, the model still works perfectly, right?

Reply:
Ah yes I missed that, I will revert it.